### PR TITLE
Add Camlistore

### PR DESCRIPTION
**Pull request by dentarg**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -233,6 +233,7 @@ CMS are a practical way to setup a website with many features. CMS often come wi
 ## File Sharing and Synchronization
 
   * [Bittorrent Sync](https://www.getsync.com/) `⊘ Proprietary` - BitTorrent Sync by BitTorrent, Inc is a proprietary peer-to-peer file synchronisation tool.
+  * [Camlistore](https://camlistore.org/) - Camlistore is a set of open source formats, protocols, and software for modeling, storing, searching, sharing and synchronizing data in the post-PC era. Data may be files or objects, tweets or 5TB videos, and you can access it via a phone, browser or FUSE filesystem. ([Demo](https://www.youtube.com/watch?v=kBCQq5hfsug), [Source code](https://github.com/camlistore/camlistore)) `Apache` `Go`
```
Comment (nodiscc) [inline README.md]: Demo must point to a live demo.
Can you remove the demo link?

**Comment by nodiscc** [comment]
Thank you @dentarg . Good to merge if you can fix the point mentioned above.

**Comment by dentarg** [comment]
@nodiscc done!

**Comment by Kickball** [comment]
I'll let it through this time, but in future if you could use shorter descriptions that would be good, this one is quite long.

Other than that thanks for the contribution!

**Comment by dentarg** [comment]
Yeah I thought about that, but saw some others with long descriptions.

Maybe you should add some guidelines about length to the contributing section?

**Comment by Kickball** [comment]
Yeah, I'll be reviewing the contributing section in the next few days, so I'll something then.

**Pull request merged**